components/frontend/src/App.js

```diff
@@ -237,27 +244,29 @@ class App extends Component {
 
     render() {
         return (
-            <AppUI
-                changed_fields={this.changed_fields}
-                dataModel={this.state.dataModel}
-                email={this.state.email}
-                openReportsOverview={() => this.openReportsOverview()}
-                handleDateChange={(date) => this.handleDateChange(date)}
-                key={this.state.report_uuid} // Make sure the AppUI is refreshed whenever the current report changes
-                lastUpdate={this.state.lastUpdate}
-                loading={this.state.loading}
-                nrMeasurements={this.state.nrMeasurements}
-                openReport={(report_uuid) => this.openReport(report_uuid)}
-                reload={(json) => this.reload(json)}
-                report_date={this.state.report_date}
-                report_uuid={this.state.report_uuid}
-                reports={this.state.reports}
-                reports_overview={this.state.reports_overview}
-                set_user={(username, email, sessionExpirationDateTime) =>
-                    this.setUserSession(username, email, sessionExpirationDateTime)
-                }
-                user={this.state.user}
-            />
+            <ThemeProvider theme={theme}>
+                <AppUI
```

Comment:
This is only indentation change right? The github viewer doesn't highlight it :(

Reply:
Yes, this delta only adds the MUI `ThemeProvider`.